www/js/app.js

```diff
@@ -48,6 +48,11 @@ angular.module('emission', ['ionic',
       StatusBar.styleDefault();
     }
     cordova.plugin.http.setDataSerializer('json');
+    // backwards compat hack to be consistent with
+    // https://github.com/e-mission/e-mission-data-collection/commit/92f41145e58c49e3145a9222a78d1ccacd16d2a7#diff-962320754eba07107ecd413954411f725c98fd31cddbb5defd4a542d1607e5a3R160
+    // remove during migration to react native
+    storage.remove("OP_GEOFENCE_CFG");
```

Comment:
Not sure I understand this.
Are we removing the geofence config because, as referenced, we now "turn OPGeofence on all the time"?

Reply:
Yes. Before this, the geofence was opt-in with a control toggle in the app (https://github.com/e-mission/e-mission-phone/commit/6f544cf793c460116106edc154a37e3052bc7a0f). The config was only "true" or "false". After field testing with it on, we decided that it was actually better and did not raise battery drain too much. So we removed it.

Reply:
The commit message has a lot more detail.